src/FlatBuffers/builder.jl

```diff
@@ -63,8 +71,8 @@ end
 Base.write(sink::Builder, o, x::Union{Bool,UInt8}) = sink.bytes[o+1] = UInt8(x)
 function Base.write(sink::Builder, off, x::T) where {T}
     off += 1
-    for (i, ind) = enumerate(off:(off + sizeof(T) - 1))
-        sink.bytes[ind] = (x >> ((i-1) * 8)) % UInt8
+    for (i, ind) in enumerate(off:(off+sizeof(T)-1))
+        sink.bytes[ind] = (x >> ((i - 1) * 8)) % UInt8
```

Comment:
What the? what's the rule here? it seems to usually take away spaces in the 1st line, but _adds_ them in the 2nd?